tests/scripts/thread-cert/border_router/test_single_border_router.py

```diff
@@ -309,6 +309,19 @@ def test(self):
         self.assertTrue(self.nodes[HOST].ping(self.nodes[ROUTER1].get_ip6_address(config.ADDRESS_TYPE.OMR)[0],
                                               backbone=True))
 
+        #
+        # Case 5. Test if the linux host is still reachable if rejoin the network.
+        #
+
+        self.nodes[HOST].disable_ether()
+        self.simulator.go(10)
+        self.nodes[HOST].enable_ether()
+        self.simulator.go(10)
```

Comment:
```suggestion
        self.simulator.go(3)
```

use smaller delay so as to make sure that RS is handled quickly ?

Reply:
I tried with `2` but failed intermediately. I think the delay comes from the way how Linux resets a network interface and it is not determined.

Reply:
Fine to me to keep it 10s.